universe/Building.h

```diff
@@ -61,13 +61,7 @@ class FO_COMMON_API Building : public UniverseObject {
 protected:
     friend class Universe;
     /** \name Structors */ //@{
-    Building() :
-        UniverseObject(),
-        m_building_type(),
-        m_planet_id(INVALID_OBJECT_ID),
-        m_ordered_scrapped(false),
-        m_produced_by_empire_id(ALL_EMPIRES)
-    {}
+    Building() {}
```

Comment:
various other cases in this PR have the {} on the next line. If it's consistent within the file then nevermind though.